src/app/pages/home/home.component.html

```diff
@@ -75,126 +101,24 @@ <h1 class="title title1">Trending Properties</h1>
-        </div>
-      </div>
-
-      <div class="row">
-        <div class="col-md-4">
-          <div class="card card1">
-            <img alt="Card image cap" class="card-img-top image1" src="./assets/img/House Architecture 7.jpg" />
-            <div class="card-body">
-              <h5 class="card-title">The Village Mall</h5>
-              <p class="card-text">
-                <i aria-hidden="true" class="fa fa-map-marker"></i> Luthuli
-                Avenue, Bugolobi.
-              </p>
-              <p class="card-text">
-                <i aria-hidden="true" class="fa fa-dollar"></i> 10,000
-              </p>
-              <a routerLink="/">
-                <button class="btn btn-warning" style="font-weight: 500">
-                  View
-                </button>
-              </a>
-            </div>
-          </div>
-        </div>
-
-        <div class="col-md-4">
-          <div class="card card1">
-            <img alt="Card image cap" class="card-img-top image1" src="./assets/img/House Architecture 8.jpg" />
-            <div class="card-body">
-              <h5 class="card-title">The Village Mall</h5>
-              <p class="card-text">
-                <i aria-hidden="true" class="fa fa-map-marker"></i> Luthuli
-                Avenue, Bugolobi.
-              </p>
-              <p class="card-text">
-                <i aria-hidden="true" class="fa fa-dollar"></i> 10,000
-              </p>
-              <a routerLink="/">
-                <button class="btn btn-warning" style="font-weight: 500">
-                  View
-                </button>
-              </a>
-            </div>
-          </div>
-        </div>
-
-        <div class="col-md-4">
+        <div class="col-md-4" *ngFor="let property of trendingProperties">
           <div class="card card1">
-            <img alt="Card image cap" class="card-img-top image1" src="./assets/img/House Architecture 9.jpg" />
+            <img
+              alt="Card image cap"
+              class="card-img-top image1"
+              src="{{ property.image_main }}"
+            />
             <div class="card-body">
-              <h5 class="card-title">The Village Mall</h5>
+              <h5 class="card-title">{{ property.title }}</h5>
               <p class="card-text">
-                <i aria-hidden="true" class="fa fa-map-marker"></i> Luthuli
-                Avenue, Bugolobi.
+                <i aria-hidden="true" class="fa fa-map-marker"></i>
+                {{ property.address.Street || property.address.State }},
+                {{ property.address.City }}.
               </p>
               <p class="card-text">
-                <i aria-hidden="true" class="fa fa-dollar"></i> 10,000
+                <b aria-hidden="true">&#8358; </b> {{property.price}}
               </p>
-              <a routerLink="/">
+              <a [routerLink]="['/properties', property.slug]">
                 <button class="btn btn-warning" style="font-weight: 500">
```

Comment:
Hey @Tevinthuku, It's important we use external files for all our styles, This will later help when we want to cache these styles over something like a CDN for first page loads.

Reply:
Done